tests/fixtures/formats/skinfolds.unsupported

```diff
@@ -0,0 +1 @@
+"ABS:3.0-PEC:3.0"
```

Comment:
what does this come from?

Reply:
when `open` understands the file extension it parses it. I added (it's commented out) another test case in the situation where you open a file that Nu doesn't support [like this one](https://github.com/nushell/nushell/blob/6824c9c7a4b7b3166284f01565fa93f6ca68260a/tests/fixtures/formats/skinfolds.unsupported)

Interestingly enough, on my machine (macOS) if the text there `ABS:3.0-PEC:3.0` is saved without double quotes and you open it (and since the test case is `open skinfolds.unsupported | echo $it`, Nu shells-out and you get a command not found.

If the text there is `"ABS:3.0-PEC:3.0"` with double quotes, it prints it in the screen. 

I wasn't sure what the desired behavior is so I left it with double quotes.

...but it breaks on Windows (the output from windows is in [the comment here](https://github.com/nushell/nushell/blob/275126d76d5126ce71cb24a03b9a3397c2e47180/tests/commands_test.rs#L68-L77)) so I left it commented out and keep everything else passing.

Reply:
> what does this come from?

Random data. No format exists for it.